docs/3-explanatory_material/explanatory_material.md

```diff
@@ -46,8 +42,8 @@ Note that if *M* is the expectation value of the system's Hamiltonian, then it i
 
 Further, an easy way to reduce the number of sub-experiments necessary is by truncating the Schmidt decomposition of |ψ> to include only some number of the bitstring states |*b*<sub>n</sub>>. However, doing so will generally lead to less accuracy in the estimation of the expectation value.
 
-## Scaling
```

Comment:
It's hard to see here, but can you just double check all the spacing between sections in these docs are consistent?

Reply:
Double spacing before `##` level, except if it comes right after a `#` title. For all other subsections single spacing.